src/core/qgstemporalnavigationobject.cpp

```diff
@@ -74,7 +75,8 @@ QgsExpressionContextScope *QgsTemporalNavigationObject::createExpressionContextS
   std::unique_ptr< QgsExpressionContextScope > scope = qgis::make_unique< QgsExpressionContextScope >( QStringLiteral( "temporal" ) );
   scope->setVariable( QStringLiteral( "frame_rate" ), mFramesPerSecond, true );
   scope->setVariable( QStringLiteral( "frame_number" ), mCurrentFrameNumber, true );
-  scope->setVariable( QStringLiteral( "frame_duration" ), mFrameDuration, true );
+  scope->setVariable( QStringLiteral( "frame_timestep" ), mFrameDuration.originalDuration(), true );
+  scope->setVariable( QStringLiteral( "frame_timestep_unit" ), mFrameDuration.originalUnit(), true );
```

Comment:
Can you revert these changes? Removing frame_duration may break some existing projects.

Reply:
sure